shared/style/dialer/dialer.css

```diff
@@ -1,5 +1,12 @@
+* {
+  /* By default do not allow any user text selection, only enable for certain
+  parts on a per-element/region area */
+  -moz-user-select: none;
+}
+
 html {
   font-size: 10px;
+  -moz-user-select: none;
```

Comment:
Maybe this should be applied on the ```body``` tag only. I don't really have a good justification here, but it makes more sense to me. The ```body``` tag contains only actual content, but the ```html``` tag contains lots of metadata (such as ```head``` content) which is not selectable to begin with.